frontend/src/components/Home/Autocomplete.tsx

```diff
@@ -15,6 +15,7 @@ import { LandlordOrApartmentWithLabel } from '../../../../common/types/db-types'
 import SearchIcon from '@material-ui/icons/Search';
 import { makeStyles } from '@material-ui/core/styles';
 import { Link as RouterLink } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
```

Comment:
Turns out `useHistory` is deprecated in the newest version of `react-router-dom`. Let's do the following: 
- Upgrade to the latest version of `react-router-dom`: in the `frontend` directory, run `yarn add react-router-dom`
- Replace `useHistory` with `useNavigation`: see this [stack overflow post](https://stackoverflow.com/a/70379483) for how to use it
- Upgrading to this version will require other changes that I've noted in other comments